HelloWorld.java

```diff
@@ -0,0 +1,9 @@
+/* This is a simple Java hello world program.
+   FileName : "HelloWorld.java". */
+class HelloWorld {
+    // Your program begins with a call to main().
+    // Prints "Hello, World" to the terminal window.
+    public static void main(String args[]) {
+        System.out.println("Hello, World");
```

Comment:
Hello, World!

Reply:
Yes, please update to print "'Hello, World!". Also, add move the source file within a reasonable package: e.g.: `com.example.hello`

Reply:
@nafSadh check now